src/views/Login/Login.vue

```diff
@@ -62,5 +62,5 @@ export default {
 
 <style scoped>
 @import "@/styles/main.css";
-@import "@/styles/login.css";
+@import "@/views/Login/login.css";
```

Comment:
same here. styles specific to the login page should be inside Login.vue. If there are styles that are used in multiple components they should be global